tools.yml

```diff
@@ -399,6 +421,8 @@ tools:
   toolshed.g2.bx.psu.edu/repos/devteam/cuffquant/cuffquant/.*:
     cores: 6
     mem: 20
+  toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS.*:
+    mem: 10
```

Comment:
For needle this needs more than 3.8GB fairly regularly for smallish inputs so I'll target this tool ID on its own. As a general rule it will only need extra memory for large input sizes.